low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -0,0 +1,155 @@
+#!/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import os
+import sys
+import errno
+import subprocess
+import shutil
+import distutils.dir_util
+from cortx.sspl.lowlevel.framework.base.sspl_constants import (REPLACEMENT_NODE_ENV_VAR_FILE, PRODUCT_NAME, SSPL_BASE_DIR,
+    file_store_config_path, PRODUCT_BASE_DIR)
+from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup.sspl_setup import Cmd as SSPLSetup
```

Comment:
sspl_post_install needs to be separately callable. You can not import wrapper

Reply:
Yes @ujjwalpl . I have made appropriate change in the latest PR.

Reply:
`[root@ssc-vm-2040 cortx-monitor]$ /opt/seagate/cortx/sspl/bin/sspl_post_install post_install -p LDR_R2
currentProduct name /opt/seagate/cortx/sspl/conf/sspl-ll.service.LDR_R2
Creating required RABBITMQ configuration....
Running via sspl service and taking key values from store factory
Executing: service rabbitmq-server status
Executing: service rabbitmq-server start
vhosts: ['Listing vhosts ...', 'name', '/', 'SSPL', '']
/usr/local/lib64/python3.6/site-packages/cryptography/hazmat/backends/openssl/hmac.py:59: UserWarning: implicit cast from 'char *' to a different pointer type: will be forbidden in the future (check that the types are as you expect; use an explicit ffi.cast() if they are correct)
  res = self._backend._lib.HMAC_Update(self._ctx, data_ptr, len(data))
/usr/local/lib64/python3.6/site-packages/cryptography/hazmat/backends/openssl/ciphers.py:114: UserWarning: implicit cast from 'char *' to a different pointer type: will be forbidden in the future (check that the types are as you expect; use an explicit ffi.cast() if they are correct)
  operation
/usr/local/lib64/python3.6/site-packages/cryptography/hazmat/backends/openssl/ciphers.py:140: UserWarning: implicit cast from 'char *' to a different pointer type: will be forbidden in the future (check that the types are as you expect; use an explicit ffi.cast() if they are correct)
  self._backend._ffi.from_buffer(data), len(data)
Users: {users}
Setting permissions for user "sspluser" in vhost "SSPL" ...
Setting tags for user "sspluser" to [administrator] ...
Creating mirrored queues...
Setting HA policy for queue actuator-req-queue
/usr/sbin/rabbitmqctl set_policy -p SSPL ha-actuator-req-queue "^actuator-req-queue$" '{"ha-mode":"all"}'
Executing: /usr/sbin/rabbitmqctl set_policy -p SSPL ha-actuator-req-queue "^actuator-req-queue$" '{"ha-mode":"all"}'
Executing: /usr/sbin/rabbitmqctl sync_queue -p SSPL actuator-req-queue
Setting HA policy for queue sensor-queue
/usr/sbin/rabbitmqctl set_policy -p SSPL ha-sensor-queue "^sensor-queue$" '{"ha-mode":"all"}'
Executing: /usr/sbin/rabbitmqctl set_policy -p SSPL ha-sensor-queue "^sensor-queue$" '{"ha-mode":"all"}'
Executing: /usr/sbin/rabbitmqctl sync_queue -p SSPL sensor-queue
Setting HA policy for queue iem-queue
/usr/sbin/rabbitmqctl set_policy -p SSPL ha-iem-queue "^iem-queue$" '{"ha-mode":"all"}'
Executing: /usr/sbin/rabbitmqctl set_policy -p SSPL ha-iem-queue "^iem-queue$" '{"ha-mode":"all"}'
Executing: /usr/sbin/rabbitmqctl sync_queue -p SSPL iem-queue
Done creating mirrored queues...`

Reply:
We can call sspl_post_install independently with no issue. @ujjwalpl @cdeshmukh